tests/unit/study/status_test_files/status_test_variables.py

```diff
@@ -0,0 +1,471 @@
+###############################################################################
+# Copyright (c) 2023, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory
+# Written by the Merlin dev team, listed in the CONTRIBUTORS file.
+# <merlin@llnl.gov>
+#
+# LLNL-CODE-797170
+# All rights reserved.
+# This file is part of Merlin, Version: 1.10.2.
+#
+# For details, see https://github.com/LLNL/merlin.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+###############################################################################
+"""This module holds variables that will be used to test against output from calls to status methods"""
+import os
+
+
+# Global path variables for files we'll need during these status tests
+PATH_TO_TEST_FILES = f"{os.path.dirname(__file__)}"
+SPEC_PATH = f"{PATH_TO_TEST_FILES}/status_test_spec.yaml"
+VALID_WORKSPACE = "status_test_study_20230717-162921"
```

Comment:
Does this mean all these empty files are here for a reason and not just a typo? If so, can we rename the directories so they look like they’re supposed to be there. The date stuff is throwing me off, as I first thought this was a simulation run

Reply:
@lucpeterson Ok I remember why I named the directories with time stamps rather than giving them "test" names: the `merlin status` command will build a list of potential studies for the user to view the status of and part of how it does this is to use a regex for the timestamp that gets appended. In order to test that this functionality works correctly we need the timestamp in the names of these directories.